.travis.yml

```diff
@@ -19,6 +22,12 @@ matrix:
       env: DB=mysql DJANGO=1.7
     - python: "3.5"
       env: DB=postgres DJANGO=1.7
+    - python: "3.3"
+      env: DB=sqlite3 DJANGO=1.9
+    - python: "3.3"
```

Comment:
I don't understand exactly what you mean. I have excluded Python 3.3 because Django 1.9 requires 2.7, 3.4+.

There are 10 envs in tox and 30 in travis (10 * 3 db backends)
- Django 1.7 (2.7, 3.3, 3.4)
- Django 1.8 (2.7, 3.3., 3.4, 3.5)
- Django 1.9 (2.7, 3.4, 3.5)

Maybe I'm missing something...